src/bin/casr-cluster.rs

```diff
@@ -197,6 +202,70 @@ fn stacktrace(path: &Path) -> Result<Stacktrace> {
     bail!("Json parse error, file: {}", path.display());
 }
 
+fn cluster_stacktraces(stacktraces: &[Stacktrace]) -> Result<Vec<u32>> {
```

Comment:
Docs + separate module: `clustering`  as name `cluster`

Reply:
moved to stacktrace.rs